src/components/formStore/FormStoreTableProvider.tsx

```diff
@@ -48,6 +57,11 @@ export function FormStoreTableProvider({
     debounceSearchMs: 1000,
     onDefaultFilters: React.useCallback((query) => {
       let defaultParameters: formStoreService.FormStoreParameters = {}
+      const localStorageParams = getParamsFromLocalStorage()
+      if (localStorageParams?.unwindRepeatableSets === true) {
```

Comment:
```suggestion
      if (localStorageParams?.unwindRepeatableSets) {
```

Reply:
`defaultParameters` is getting overwritten below.

Reply:
I didn't want to do a truthy match in case the value contained something like "dog" - not that it can unless the user goes in and changes it.  But I felt safer this way

Reply:
@Zaxist true, will fix